cmd/neofs-node/attributes.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 
 	"github.com/nspcc-dev/locode-db/pkg/locodedb"
+	"github.com/nspcc-dev/neofs-api-go/v2/netmap"
```

Comment:
I need `SetAttributes`, to clear all the attributes, and `GetAttributes`, so that I don't have to write a lot of code. I thought if they weren't there, then there was a reason for that. So I can add them?

Reply:
feel free to add. Recent similar [addition](https://github.com/nspcc-dev/neofs-sdk-go/commit/104b47995237191ea0a38a51637e8c39e8f92329). U may add `SetAttributes` or `ResetAttributes`

Reply:
And now, how do I use what I've extended? Or will it need to be fixed later, after updating the SDK version? Meanwhile, should I make an issue?

Reply:
`go get github.com/nspcc-dev/neofs-sdk-go@master`